src/main/python/treccar/augment_collection_with_predictions.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+"""
+Anserini: A Lucene toolkit for replicable information retrieval research
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import os
+import argparse
+from trec_car_classes import *
+
+
+def convert_collection(args):
+    print('Converting collection...')
+
+    predictions_file = open(args.predictions)
+    file_index = 0
+    with open(args.collection_path, 'rb') as f:
+
```

Comment:
seems like there are too many empty lines? here and below...

Reply:
Fixed